src/pages/ProfileQuestionPage/Answer/index.tsx

```diff
@@ -1,19 +1,131 @@
-// components/Answer.tsx
-import { Box } from '@chakra-ui/react'
+import { useEffect, useRef, useState } from 'react'
+import { BiError, BiTrash } from 'react-icons/bi'
 
+import { Box, Button, useDisclosure } from '@chakra-ui/react'
+import { useMutation } from '@tanstack/react-query'
+
+import { queryClient } from '@/api/instance'
+import {
+  DeleteProfileAnswerRequest,
+  deleteProfileAnswer,
+  useGetProfileAnswer,
+} from '@/api/services/profile/profileQuestion.api'
 import { ChatBox } from '@/components/ChatBox'
+import { Loading } from '@/components/Loading'
+import { AlertModal } from '@/components/Modal/AlertModal'
+import { ConfirmModal } from '@/components/Modal/ConfirmModal'
+import ErrorPage from '@/pages/ErrorPage'
+import { useSelectedQuestionStore } from '@/stores/selected-question'
 import { ChatItem } from '@/types'
 
+import formatDate from './formatDate'
+
 interface AnswerProps {
-  dummyDataList: ChatItem[]
+  userId: number
+  isMyPage: boolean
 }
 
-const Answer: React.FC<AnswerProps> = ({ dummyDataList }: AnswerProps) => {
+const Answer: React.FC<AnswerProps> = ({ userId, isMyPage }: AnswerProps) => {
+  const questionId = useSelectedQuestionStore((state) => state.questionId)
+  const [selectDeleteAnswerId, setSelectDeleteAnswerId] = useState<
+    number | null
+  >(null)
+  const deleteAlert = useDisclosure()
+  const errorAlert = useDisclosure()
+
+  const { mutate: deleteAnswer } = useMutation<
+    void,
+    Error,
+    DeleteProfileAnswerRequest
+  >({
+    mutationFn: ({ deleteAnswerId }: DeleteProfileAnswerRequest) =>
```

Comment:
```suggestion
mutationFn: (deleteAnswerId) => deleteProfileAnswer(deleteAnswerId),
```

여기도 타입추론이 자동으로 되어서 타입 지정 안해줘도 됩니다!